include/ublksrv.h

```diff
@@ -914,6 +914,13 @@ extern int ublksrv_process_io(const struct ublksrv_queue *q);
 extern int ublksrv_complete_io(const struct ublksrv_queue *q, unsigned tag, int res);
 /** @} */ // end of ublksrv_queue group
 
+/**
+ * Reap events received via io_uring.
+ *
+ * @param r the ring pointer for io_uring
+ */
+extern int ublksrv_reap_events_uring(struct io_uring *r);
```

Comment:
Can you name it as ublksrv_queue_reap_events(struct ublksrv_queue *)?
Then we can avoid to add io_uring into interface, meantime ublksrv_reap_events_uring() can be kept as inline.

Thanks,

Reply:
Ok, done.